src/plugins/vis_type_xy/public/components/xy_settings.tsx

```diff
@@ -70,6 +72,27 @@ type XYSettingsProps = Pick<
   legendPosition: Position;
 };
 
+function getValueLabelsStyling(isHorizontal: boolean) {
+  const VALUE_LABELS_MAX_FONTSIZE = 15;
+  const VALUE_LABELS_MIN_FONTSIZE = 10;
+  const VALUE_LABELS_VERTICAL_OFFSET = -10;
+  const VALUE_LABELS_HORIZONTAL_OFFSET = 10;
+
+  return {
+    displayValue: {
+      fontSize: { min: VALUE_LABELS_MIN_FONTSIZE, max: VALUE_LABELS_MAX_FONTSIZE },
+      fill: { textInverted: true, textBorder: 2 },
+      alignment: isHorizontal
+        ? {
+            vertical: VerticalAlignment.Middle,
+          }
+        : { horizontal: HorizontalAlignment.Center },
+      offsetX: isHorizontal ? VALUE_LABELS_HORIZONTAL_OFFSET : 0,
+      offsetY: isHorizontal ? 0 : VALUE_LABELS_VERTICAL_OFFSET,
```

Comment:
These values should all come from the `theme` object. This will override all of these properties and ignore the value defined in the `theme`.

You should be able to add these overrides via the `Settings.theme` prop already. What is the use case for this?

Reply:
As I understood we should use defaults from Lens for values styles. These values which I found in Lens we should use in plugin. As we already have 'themeOverrides' variable I think we should add these defaults there. For convenience I created a separate function for this.

Reply:
Wow, sorry 😐. When I looked at this PR I thought this was done in [@elastic/charts](https://github.com/elastic/elastic-charts). Just ignore me and my comment this is perfect!